src/common/signalk/SKJSONVisitor.cpp

```diff
@@ -30,16 +30,47 @@
 
 #include "SKJSONVisitor.h"
 
+void SKJSONVisitor::processSource(const SKSource &source, JsonObject &sourceObject) {
+  sourceObject["label"] = source.getLabel().c_str();
+
+  switch (source.getInput()) {
```

Comment:
I got the impression that "talker" was supposed to identify the source of Signal K data itself (i.e. always the kbox), and not who originally generated the NMEA data, so that you could tell multiple sK streams apart? I probably misunderstood that, in which case this is fine :)